src/screens/ProfileScreen/index.js

```diff
@@ -725,7 +724,7 @@ const ProfileScreen = ({route}) => {
         onClose={onHeaderOptionClosed}
         onDeleteClicked={onDeletePost}
       />
-    </SafeAreaProvider>
+    </View>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Same comment as above regarding the replacement of `SafeAreaProvider` with `View`.
```diff
- </View>
+ </SafeAreaProvider>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->